config/nav.yml

```diff
@@ -202,15 +222,6 @@ nav:
           - Available Channels: eventing/channels/channels-crds.md
           - Subscriptions: eventing/channels/subscriptions.md
         - Sugar Controller: eventing/sugar/README.md
-        - Brokers:
-          - About Brokers: eventing/broker/README.md
-          - Using the Event registry: eventing/event-registry.md # does this make sense here since it mentions being used with brokers and triggers?
-          - Creating a Broker: eventing/broker/create-mtbroker.md
-          - Triggers: eventing/broker/triggers/README.md
-          - Broker configuration example: eventing/broker/example-mtbroker.md
-          - Knative Kafka Broker: eventing/broker/kafka-broker/README.md
-          - RabbitMQ Broker: eventing/broker/rabbitmq-broker/README.md
-        - Accessing CloudEvent traces: eventing/accessing-traces.md
```

Comment:
Seems like you are moving this section under `Resources`. What about the other resources, like Channels, sources, etc... ?

Reply:
Planning to move them all over time but it's not realistic to do it all in one PR. I've moved Revisions as well already for Serving and will keep adding things as I get time. There's a bit of cleanup required every time I do one of these.

Reply:
They're also under Concepts - the idea is to make it easy for users to find info about what the resources and related concepts are, how and why things are used, etc..